src/main/java/duke/command/GetCommand.java

```diff
@@ -0,0 +1,33 @@
+package duke.command;
+
+import duke.Storage;
+import duke.task.TaskList;
+import duke.DukeException;
+import duke.Parser;
+import duke.Ui;
+
+/**
+ * Represents command that is specific to the get command.
+ */
+public class GetCommand extends Command {
```

Comment:
May I ask what is GetCommand for? Correct me if I'm wrong but assuming this is for finding task, perhaps you might want to consider renaming this class to FindCommand to avoid confusion. Since get is a bit ambiguous (can also be interpreted as a verb).